.github/workflows/release.yml

```diff
@@ -4,9 +4,13 @@ on:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9]+)?"
 
+permissions:
+  contents: read
+  id-token: write
+
 env:
-  gcp-project-id: 118585658141
-  service-account: gh-ci-optable-web-sdk
+  workload_identity_provider: ${{ secrets.WORKLOAD_IDENTITY_PROVIDER }}
```

Comment:
We can put the value directly here as the workload_identity_provider is not sensitive.